dataframe-excel/README.md

```diff
@@ -0,0 +1,8 @@
+## :dataframe-excel
+
+This module, published as `dataframe-excel`, contains all logic and tests for DataFrame to be able to work with
+Excel files.
```

Comment:
I usually mention `xls` and `xlsx` specifically

Reply:
good idea, you can also produce these filetypes with google sheets for instance